### ISSUE TITLE
adblocker blocks craigslist images

### ISSUE THREAD

**Issue by cryptobird** [PalmerAL/min#125]

**Comment by PalmerAL** [comment]
It doesn't look like this is caused by the adblocker:

<img width="866" alt="screen shot 2016-05-05" src="https://cloud.githubusercontent.com/assets/10314059/15062081/28e45140-1302-11e6-9962-ad325d607d6b.png">

The images still shouldn't get blocked though, even if they are served over HTTP.

**Comment by cryptobird** [comment]
@PalmerAL Yeah, do you want me to report this... with them? however one does that?

**Comment by PalmerAL** [comment]
I guess you could... it's probably a Min issue though, since it works fine in Chrome, Firefox, Safari, etc.

Craigslist isn't the only website that has this problem, either–it happens on iTunes preview links as well (example: https://itunes.apple.com/us/app/slither.io/id1091944550).

**Comment by cryptobird** [comment]
Ah, woops, I misread. I thought you said it looked like it was being caused by the adblocker.

**Comment by dbfreem** [comment]
I found this site https://www.mixedcontentexamples.com/ to test mixed content and all of the images I checked showed up correctly. Anyone have thoughts on if this issue still exist?
